lib/github_changelog_generator/parser.rb

```diff
@@ -116,6 +116,9 @@ def self.setup_parser(options)
         opts.on("--between-tags  x,y,z", Array, "Change log will be filled only between specified tags") do |list|
           options[:between_tags] = list
         end
+        opts.on("--exclude-tags-filter [REGEX]", "Apply a regular expression on tag names so that they can be excluded, for example: --exclude-tags-filter \".*\+\d{1,}\" ") do |last|
+          options[:exclude_tags_filter] = last
```

Comment:
Late feedback: why is this block parameter called `last` (where the others are called `list`)?